explore/src/main/scala/explore/constraints/ConstraintsSummaryTable.scala

```diff
@@ -106,7 +124,7 @@ object ConstraintsSummaryTable extends TableHooks:
               ctx.pageUrl(AppTab.Constraints, props.programId, Focused.singleObs(obsId))
 
             List(
-              column("edit", ConstraintGroup.obsIds.get)
+              column(ColumnId("edit"), ConstraintGroup.obsIds.get)
```

Comment:
Can't you use `EditColumnId` that is defined above?

Reply:
Yup, missed this, thank you.